grails-app/i18n/messages.properties

```diff
@@ -18,6 +18,32 @@ general.list.orderBy.label = Order by
 general.list.sortBy.label = Sort by
 general.loading = Loading
 
+# Download
+download.title = Download
+download.terms.01 = By downloading this content you are agreeing to use it in accordance with the Atlas of Living Australia
```

Comment:
Maybe eElurikkus or smth instead of Australia

Reply:
😂

Reply:
god dammit, still copied the wrong message...